### PR TITLE
Update for v0.4.8

### PR DESCRIPTION
**Pull request by pwelch**
## Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ ] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/sigstore-prober/Chart.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ annotations:
   artifacthub.io/license: Apache-2.0
   artifacthub.io/images: |
     - name: sigstore-prober
-      image: ghcr.io/sigstore/scaffolding/prober@sha256:d16b6489228e30cccda4395086148375f2f1aff395aa0b66dc54cfd46cf6309f
+      image: ghcr.io/sigstore/scaffolding/prober@sha256:79a270c22743f49a7be56961d95642a7be8e5799bfd3a0f2f7752ad10bf28d2c # v0.4.8
```
Comment (vaikas) [inline charts/sigstore-prober/Chart.yaml:24]: I think we actually want this sha instead (which is the non-arch specific one):
sha256:f2984afbeff4e51963d6103c5bdf69f6fb1750bebbc584e22cecf0c89a4afe27

https://github.com/sigstore/scaffolding/pkgs/container/scaffolding%2Fprober/42671649?tag=v0.4.8

I think the one above is for linux/s390x (if you click on the OS/Arch tab that's the one)

Also checked that just re-running the release created the same sha as expected :)

Comment (pwelch) [reply charts/sigstore-prober/Chart.yaml]: Ah thanks for catching that! I updated and rebased the branch.

**Review by hectorj2f — COMMENTED** [review]
@pwelch You should bump the chart version.

**Comment by pwelch** [comment]
@hectorj2f yeah, just noticed and doing now 😅

**Review by priyawadhwa — APPROVED** [review]
nice!!

**Pull request merged**